motoko/fileupload/src/fileupload/types.mo

```diff
@@ -0,0 +1,53 @@
+module {
+
+    public type HeaderField = (Text, Text);
+
+    public type Chunk = {
+        batch_name : Text;
+        content  : [Nat8];
```

Comment:
Its much more efficient if the data chuncks are imported as Blobs rather than `[Nat8]`, since a [Nat8] takes 4 x the space. Both types are importable, exportable as  the same, compact Candid `vec nat8`, but `[Nat8]` takes much more space in Motoko memory than `Blob`.